src/content-script/index.mjs

```diff
@@ -32,8 +32,17 @@ async function run(question) {
   port.postMessage({ question });
 }
 
+function endsWithQuestionMark(question) {
+  return (
+    question.endsWith("?") ||  // ASCII
+    question.endsWith("？") || // Chinese/Japanese
+    question.endsWith("؟") || // Arabic
+    question.endsWith("⸮") // Arabic
```

Comment:
Your probably need to add the underline question mark for Spanish "¿"

Reply:
@alhazmy13 Thanks for the advice! However, because Spanish question also ends with "?", there's no need to check its beginning "¿".